docs/content/docs/slsa/frsca-slsa.md

```diff
@@ -0,0 +1,117 @@
++++
+title = "How FRSCA is meeting SLSA levels 1-3"
+description = "A document describing how FRSCA is meeting SLSA compliance, specifically levels 1-3"
+date = 2023-02-07T08:20:00+00:00
+updated = 2023-02-07T08:20:00+00:00
+draft = false
+weight = 20
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = false
++++
+
+## Reminder on SLSA levels
+
+SLSA levels at a high level are as follows (table from [https://slsa.dev/spec/v0.1/levels#summary-of-levels](https://slsa.dev/spec/v0.1/levels#summary-of-levels)):
+
+| **Level** | **Description**                        | **Example**                                           |
+| --------- | -------------------------------------- | ----------------------------------------------------- |
+| 1         | Documentation of the build process     | Unsigned provenance                                   |
+| 2         | Tamper resistance of the build service | Hosted source/build, signed provenance                |
+| 3         | Extra resistance to specific threats   | Security controls on host, non-falsifiable provenance |
+| 4         | Highest levels of confidence and trust | Two-party review + hermetic builds                    |
+
+The full requirements table can be found here:
+
+[Requirements](https://slsa.dev/spec/v0.1/requirements#summary-table)
+
+## sample-pipeline
+
+```other
+frsca/examples/sample-pipeline
+```
+
+For the purposes of this document, we're going to be comparing SLSA requirements to what we are deploying via the sample-pipeline.  Please read `frsca/examples/sample-pipeline/README.md` for information on setting up FRSCA and the sample pipeline.
+
+# SLSA 1 Requirements
+
+1. Build - [Scripted build](https://slsa.dev/spec/v0.1/requirements#scripted-build) **✅**
+2. Provenance - [Available](https://slsa.dev/spec/v0.1/requirements#available) ✅
+
+`sample-pipeline` meets both of these requirements.  The build is scripted, which is evident in following individual steps from the `Makefile` down through the various shell scripts which setup the FRSCA environment and trigger the build via Tekton Pipelines.
+
+Because we're using Tekton Chains in conjunction with Tekton Pipelines, provenance is created and available, satisfying the last requirement of SLSA 1.
+
+To prove, once the pipeline completes:
+
+```other
+> crane ls "$(echo -n ${IMAGE_URL} | sed 's|:[^/]*$||')"            
+                                                                                                                                                                                            
+0066d00de427d12b9a14e56f02f302031d9c40f3
+sha256-e6dc8ea1ff666893462b64d997d496af8e69e905f2eeaf2ab7ec1fd565921d46.att
+sha256-e6dc8ea1ff666893462b64d997d496af8e69e905f2eeaf2ab7ec1fd565921d46.sbom
+sha256-e6dc8ea1ff666893462b64d997d496af8e69e905f2eeaf2ab7ec1fd565921d46.sig
+```
+
+# SLSA 2 Requirements
+
+1. Source - [Version controlled](https://slsa.dev/spec/v0.1/requirements#version-controlled) ✅*
```

Comment:
At the end of this line, there is an asterisk, but it is not pointing to anything.

Reply:
I placed an asterisk to just denote that it's something that the FRSCA examples cover, but it's ultimately up to the end user to ensure their source requirements are fulfilled.  If it's confusing I'm happy to just take it off.

Reply:
It makes sense once you explain it 😃 Maybe you could just mention that explanation at the bottom of the list (or wherever you think it would make sense)?